joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -233,8 +233,9 @@ trait AstCreatorHelper(implicit withSchemaValidation: ValidationMode) { this: As
     "/="  -> Operators.assignmentDivision,
     "%="  -> Operators.assignmentModulo,
     "**=" -> Operators.assignmentExponentiation,
-    // Strictly speaking, `a ||= b` means `a || a = b`, but I reckon we wouldn't gain much representing it that way.
-    "||=" -> Operators.assignmentOr,
```

Comment:
Was this actually invalid Ruby code?

Reply:
This is detected as a separate entity by the parser - a unique structure. So this is valid, but the visitor for it lies elsewhere in the `RubyJsonNodeCreator` and it never pings this map